sapi/src/lib.rs

```diff
@@ -8,31 +8,52 @@ use reqwest::hyper_011::header::{Accept, ContentType, Headers};
 use serde::{Deserialize, Serialize};
 use serde_json::{json, Value};
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct SapiManifests {
```

Comment:
Will this require maintenance going forward as the API changes?

Reply:
It is an unfortunate situation we are in.  The schema is not very well defined, but there are some fields that are always present. I went off of sapi's bucket creation as linked here:
https://github.com/joyent/sdc-sapi/blob/a4209a1eabb9d8db2170617195a08516bba845f5/lib/server/stor/moray.js#L183-L263

If that changes then we will need to update this.  I'll make a comment to that effect.